CMakeLists.txt

```diff
@@ -370,6 +370,10 @@ elseif(GNU_GCC OR LLVM_CLANG)
   endif()
 endif()
 
+set(CMAKE_C_VISIBILITY_PRESET hidden)
+set(CMAKE_CXX_VISIBILITY_PRESET hidden)
+set(CMAKE_VISIBILITY_INLINES_HIDDEN TRUE)
```

Comment:
Is this still required since you fixed the protobuf package?

Reply:
This "fixes" the visibility of the Mixxx functions. Works for us, because we are no shared library.

Reply:
So then it is irrelevant for Mixxx. We should not be adding code that serves no purpose.

Reply:
Without these lines we receive linker warnings.

Reply:
What are the linker warnings for? Vendored libraries?

Reply:
See: https://github.com/mixxxdj/mixxx/runs/3395023796

Reply:
Huh, okay. I'm not sure what to make of that. I think it's fine to leave this code but there should be a comment explaining why it is there.